netdata/netdata
# Log ACLK cloud commands to access.log

##### Summary
Logs ACLK requests from the cloud to the access.log
##### Component Name
database
aclk

##### Test Plan
- Start a cloud enabled / claimed agent
- Login to the cloud
- Go to the local agent and observe the `access.log`
   - You should see entries similar to 
      ```
      2021-03-01 15:42:55: 0: 196758 '[ACLK]:2' 'DATA' (sent/all = 3340/8971 bytes -63%, prep/sent/total = 0.04/1.01/1.05 ms) 200 '/api/v1/data'
      2021-03-01 15:42:55: 0: 196756 '[ACLK]:0' 'DATA' (sent/all = 3780/8299 bytes -54%, prep/sent/total = 0.02/0.41/0.43 ms) 200 '/api/v1/data'
       ```
This indicates that the /api/v1/data command is issued via ACLK and is processed by query thread 2 and query thread 0 respectively.

##### Additional Information
